ohttp/src/lib.rs

```diff
@@ -116,6 +116,23 @@ impl KeyConfig {
         })
     }
 
+    /// Derive a configuration for the server side from input keying material
+    /// # Panics
+    /// If the configurations don't include a supported configuration.
+    #[cfg(feature = "rust-hpke")]
+    pub fn derive(key_id: u8, kem: Kem, mut symmetric: Vec<SymmetricSuite>, ikm: &[u8]) -> Res<Self> {
```

Comment:
I think that you want this to be unconditional and just provide a `derive_key_pair` function on the NSS side as well.  You can stub that and have it return `Err(Error::Unsupported)` or something similar to that.

Reply:
That's better, yeah.